logstash-core-event-java/src/jmh/java/com/logstash/ThreadStateBase.java

```diff
@@ -0,0 +1,106 @@
+package com.logstash;
+
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.Map;
+
+public abstract class ThreadStateBase {
```

Comment:
I am not sure about all of the necessity of this class, maybe missing something about how Kyro serializes things in a multithreaded manner?